ci-operator/jobs/open-cluster-management/security-middleware/open-cluster-management-security-middleware-main-postsubmits.yaml

```diff
@@ -0,0 +1,36 @@
+postsubmits:
```

Comment:
Why there is a postsubmit that runs the same test as the presubmit?

Reply:
This is not a test, it is an action. Sonar scans are run on PRs during the presubmit phase to check code quality before the PR is merged (and block the PR if it doesn't meet its quality gate. The scan results are published at sonarcloud.io.

After the PR is merged, the Sonar scan has to be run again, but this time on the branch that was just merged into. Again the results are published to sonarcloud.io.